SoftLayer/CLI/vlan/list.py

```diff
@@ -35,7 +38,10 @@
               show_default=True)
 @environment.pass_env
 def cli(env, sortby, datacenter, number, name, limit):
-    """List VLANs."""
+    """List VLANs.
+
+    Note: In field Pod, if add (*) indicated that closed soon
```

Comment:
I think add color this message

Reply:
Cannot add color to a specific text for different reasons

Reply:
Good news is you CAN add colors since I moved to Rich Text module in python.

https://rich.readthedocs.io/en/stable/markup.html
```suggestion
    Note: A * Indicates a POD is closing soon. Ex: [red] Pod01* [/red]
```

![image](https://user-images.githubusercontent.com/7408017/181844443-ed738878-516b-41bc-8609-321fc6de3329.png)

Reply:
Done